rust-runtime/smithy-http/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "Apache-2.0"
 
 [features]
 bytestream-util = ["tokio/fs", "tokio-util/io"]
+sleep-tokio = ["tokio/time"]
```

Comment:
I think we probably don't want this in smithy-http. Also, since this doesn't actually bring in the runtime, this isn't sufficient to actually work.